### PR TITLE
[Merged by Bors] - chore(.github/workflows): add label after maintainer merge

### PR DESCRIPTION
**Pull request by jcommelin**
By automatically adding a label,
it is easier to find PRs that have been maintainer merged
but slipped through some crack.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: .github/workflows/maintainer_merge_comment.yml [hunk 1]
```diff
@@ -40,3 +40,10 @@ jobs:
           issue-number: ${{ github.event.issue.number }}
           body: |
             🚀 Pull request has been placed on the maintainer queue by ${{ github.event.comment.user.login }}.
+      - name: Add label to PR
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const { owner, repo, number: issue_number  } = context.issue;
```
Comment (Ruben-VandeVelde) [inline .github/workflows/maintainer_merge_comment.yml:48]: Nitpicking:
```suggestion
            const { owner, repo, number: issue_number } = context.issue;
```

File: .github/workflows/maintainer_merge_review.yml [hunk 2]
```diff
@@ -38,3 +38,10 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body: |
             🚀 Pull request has been placed on the maintainer queue by ${{ github.event.review.user.login }}.
+      - name: Add label to PR
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const { owner, repo, number: issue_number  } = context.issue;
```
Comment (Vierkantor) [inline .github/workflows/maintainer_merge_review.yml:46]:
```suggestion
            const { owner, repo, number: issue_number } = context.issue;
```

File: .github/workflows/maintainer_merge_review_comment.yml [hunk 3]
```diff
@@ -38,3 +38,10 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body: |
             🚀 Pull request has been placed on the maintainer queue by ${{ github.event.comment.user.login }}.
+      - name: Add label to PR
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const { owner, repo, number: issue_number  } = context.issue;
```
Comment (Vierkantor) [inline .github/workflows/maintainer_merge_review_comment.yml:46]:
```suggestion
            const { owner, repo, number: issue_number } = context.issue;
```

**Review by Vierkantor — APPROVED** [review]
Looks good to me, did you test this? Or do we test on production in the assumption it's not too bad if things break? :)

**Comment by jcommelin** [comment]
@Vierkantor I was hoping we could test this one in production. Note that this is basically a copy-paste from similar functionality that we already have for other labels. So I would guess 99% everything just works, 1% something silly goes wrong without any bad consequences.

**Comment by Vierkantor** [comment]
Yeah, since this is just for reviewers/maintainers I'm willing to take the risk.

bors r+

**Pull request closed without merging**